packages/near-membrane-base/src/intrinsics.ts

```diff
@@ -146,13 +151,9 @@ export function getFilteredEndowmentDescriptors(endowments: object): PropertyDes
         // will be ignored if present in the endowments object.
         // TODO: what if the intent is to polyfill one of those
         // intrinsics?
-        if (!ReflectApply(SetProtoHas, ESGlobalKeys, [key])) {
+        if (!isIntrinsicGlobalName(key)) {
             to[key] = ReflectGetOwnPropertyDescriptor(endowments, key)!;
         }
     }
     return to;
 }
-
-export function isIntrinsicGlobalName(key: PropertyKey): boolean {
```

Comment:
this is not longer needed outside of this file... moving it up and removing the export part of it.